### PR TITLE
Updates to HCL 2.0

### PR DESCRIPTION
**Pull request by ksatirli**
This PR cleans up the development tooling and updates all code to HCL 2.0

### CODE REVIEW

**Comment by ksatirli** [comment]
@kibertoad can you check this when you have some time?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+Additionally, the following variables are generated as [locals](https://www.terraform.io/docs/configuration/locals.html):
+
+| key            | value                                                  |
+|----------------|--------------------------------------------------------|
+| `zone_name`    | `data.aws_route53_zone.zone.name`                      |
+| `domain_proof` | "keybase-site-verification=${var.verification_record}" |
+
+### Module Outputs
+
+Available outputs are listed below, along with their description
```
Comment (kibertoad) [inline README.md]: `:` might make sense here, to make this consistent with other place

File: README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+| output         | description                                               |
+|----------------|-----------------------------------------------------------|
+| `domain_proof` | interpolated value of `aws_route53_record.ownership.name` |
```
Comment (kibertoad) [inline README.md]: Is naming correct here? Should it be `domain_proof`/`ownership.name`?

File: outputs.tf [hunk 3]
```diff
@@ -0,0 +1,9 @@
+output "zone_name" {
+  value       = local.zone_name
+  description = "interpolated value of `local.zone_name`"
+}
+
+output "domain_proof" {
+  value       = aws_route53_record.domain_proof.name
+  description = "interpolated value of `aws_route53_record.domain_proof.name`"
```
Comment (kibertoad) [inline outputs.tf:8]: there seems to be a mismatch with documentation here

File: README.md (hunk 2 above)
Comment (ksatirli) [reply README.md]: Updated!

File: outputs.tf (hunk 3 above)
Comment (ksatirli) [reply outputs.tf:8]: I agree; I'm now using `segmentio/terraform-docs` to prevent these issues going forward

**Review by kibertoad — APPROVED** [review]

**Pull request merged**